### PR TITLE
[fix] [sl] Slovenian translation fix

### PR DESCRIPTION
**Pull request by peterkokot**
This ads Slovenian translation and fixes BackBee English Twig template.

### CODE REVIEW

File: projects/sl/backbee.html.twig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% block description %}
+    <p>
+
+        BackBee je odprto kodni CMS načrtovan za revolucijo uporabniške izkušnje.
+        Uporablja arhitekturo "urejanje na strani", kar naredi distribucijo vsebine intiutivno in varno
```
Comment (javiereguiluz) [inline projects/sl/backbee.html.twig]: Typo? `intiutivno` -> `intuitivno`.

Comment (peterkokot) [inline projects/sl/backbee.html.twig]: :flushed: You're right. Sorry, fixed. Thanks for noticing.

**Comment by javiereguiluz** [comment]
:+1: thanks @peterkokot!

**Comment by mickaelandrieu** [comment]
thank you :)

**Comment by peterkokot** [comment]
Ping @fabpot: any chance to merge these updates one as well? Thank you in advance.

**Comment by javiereguiluz** [comment]
@peterkokot I'm sorry it took us so long to merge your changes. Thanks a lot for the fixes and new translations!

**Pull request closed without merging**